### PR TITLE
msx1_cart.xml: Software list additions

### PR DESCRIPTION
**Pull request by wilbertpol**
New working software list items
-------------------------------
Warroid (Japan, alt) [file-hunter]
Water Driver (Spain, alt) [file-hunter]
Senjou no Ookami (Japan, alt) [file-hunter]
Word Processor (United Kingdom) [file-hunter]
Yellow Submarine (Japan, Jast) [file-hunter]
Yie Ar Kung-Fu II - The Emperor Yie-Gah (Korea) [file-hunter]
Youkai Tantei Chimachima (Japan, alt) [file-hunter]
Yogoetamjeong (Korea) [file-hunter]
Youkai Yashiki (Japan, alt) [file-hunter]
Young Artist (Arab, v1.18) [file-hunter]
Chou Senshi Zaider - Battle of Peguss (Korea) [file hunter]
Zanac A.I. - 2nd Version (Japan, alt 2) [file-hunter]
Zenji (Japan, alt 1) [file-hunter]
Zexas Limited (Japan, alt 3) [file-hunter]
Westen House (English, v1.3.1) [santiontanon]
Westen House (Spanish, v1.3.1) [santiontanon]
Westen House (Portuguese, v1.3.1) [santiontanon]
Westen House (English, v1.3) [santiontanon]
Westen House (Spanish, v1.3) [santiontanon]
Westen House (English, v1.2) [santiontanon]
Where Is It? The Quest for the 10th Island [MSXDev]
Where Is It? The Quest for the 10th Island (proto) [JMeric]
Wing Warriors (Reprosoft) [file-hunter]
Wing Warriors (MSXDev) [MSXDev]
Witch Day (English, v1.2) [MSXDev]
Dia de Brujas (Spanish, v1.2) [MSXDev]
Witch Day (English, v1.0) [fiile-hunter]
Dia de Brujas (Spanish, v1.0) [file-hunter]
Witches [cobinee]
Word Invaders [MSXDev]
Wörd! (English, v1.1) [MSXDev]
Wörd! (Spanish, v1.1) [MSXDev]
Wörd! (Dutch, v1.1) [MSXDev]
Wörd! (Swedish, v1.1) [MSXDev]
Wörd! (English, v1.0) [file-hunter]
Wörd! (Spanish, v1.0) [file-hunter]
Wörd! (Dutch, v1.0) [file-hunter]
Wörd! (Swedish, v1.0) [file-hunter]
X0rz (v3) [MSXDev]
X0rz (v1) [MSXDev]
XOR 2021 [MSXDev]
XRacing (v1.0.5) [santiontanon]
XRacing (v1.0.4) [santiontanon]
XRacing (v1.0.3) [file-hunter]
XRacing (v1.0.2) [santiontanon]
XRacing (v1.0.1) [santiontanon]
XRacing (v1.0) [santiontanon]
XSpelunker (v1.4.3) [santiontanon]
XSpelunker (alt) [file-hunter]
XSpelunker (v1.4.2) [MSXDev]
XSpelunker (v1.4.1) [santiontanon]
XSpelunker (v1.4) [santiontanon]
XSpelunker (v1.3) [santiontanon]
XSpelunker (v1.2) [santiontanon]
XSpelunker (v1.1) [santiontanon]
XSpelunker (v1.0) [santiontanon]
Yab Yum (Netherlands, alt) [file-hunter]
Yab Yum (Netherlands, alt 2) [file-hunter]
Yahtzee [MSXDev]
Yazzie [RetroSouls]
Yazzie (alt, older?) [file-hunter]
Yazzie Remastered [MSXDev]
Z (English) [cobinee]
Z (Korean) [file-hunter]
Z2  (Japan) [cobinee]
z80Rogue [file-hunter]
ZERO and the Castle of Infinite Sadness (Matra) [file-hunter]
ZERO and the Castle of Infinite Sadness (MSXDev) [file-hunter]
ZERO and the Castle of Infinite Sadness (MSXDev, with bug) [file-hunter]
Zombie Calavera Prologue [file-hunter]
Zombie Incident (v1.2) [MSXDev]
Zombie Incident (v1.1) [file-hunter]
Zombie Incident (v1.0) [file-hunter]
Zombie Near (v1.1) [MSXDev]
Zombie Near (v1.0) [file-hunter]
Zone TNT [MSXDev]
Wild Cat (Japan, cas2crt conversion) [file-hunter]
Xadrez (Brazil, cas2crt conversion) [file-hunter]
XVM (Japan, flop2crt conversion) [file-hunter]
Zeta 2000 (Japan, cas2crt conversion) [file-hunter]

New NOT_WORKING software list additions
------------------------------------------
X & Y (Arab) [file-hunter]

### CODE REVIEW

File: hash/msx1_cart.xml [hunk 1]
```diff
@@ -13003,6 +13075,33 @@ kept for now until finding out what those bytes affect...
...
+			<feature name="start_page" value="1"/>
+			<dataarea name="rom" size="0x8000">
+				<rom name="yokai tanken chimachima (japan, alt).rom" size="0x8000" crc="39460ce0" sha1="a574bb9b668ae3bf40a0d57d9aed3cfa19b559bb" />
+			</dataarea>
+		</part>
+	</software>
+
+	<!-- No photographic proof of a physical release. -->
+	<software name="chimak" cloneof="chima">
+		<description>Youkai Tantei Chima Chima (Korea)</description>
```
Comment (cuavas) [inline hash/msx1_cart.xml:13094]: Does this have a Korean title screen?  The Korean reading of the ideographs is “요괴탐정”, i.e. “Yogoetamjeong” in RR.

Comment (wilbertpol) [reply hash/msx1_cart.xml:13094]: Yes, it has Korean on the title screen:
Screenshot 2023-04-11 at 21 34 46

File: hash/msx1_cart.xml [hunk 2]
```diff
@@ -13003,6 +13075,33 @@ kept for now until finding out what those bytes affect...
 		</part>
 	</software>
 
+	<software name="chimaa">
+		<description>Youkai Tantei Chima Chima (Japan, alt)</description>
+		<year>1985</year>
+		<publisher>Bothtec</publisher>
+		<info name="serial" value="MK-7401" />
+		<info name="alt_title" value="妖怪探偵ちまちま" />
```
[the remark this answers is not in the document]
Comment (wilbertpol) [reply hash/msx1_cart.xml]: Removed the space from clone and parent

File: hash/msx1_cart.xml (hunk 1 above)
Comment (wilbertpol) [reply hash/msx1_cart.xml]: Updated the description and added alt_title.

**Pull request merged**